Source/FieldSolver/SpectralSolver/SpectralAlgorithms/GalileanAlgorithm.H

```diff
@@ -0,0 +1,33 @@
+#ifndef WARPX_GALILEAN_ALGORITHM_H_
+#define WARPX_GALILEAN_ALGORITHM_H_
+
+#include <SpectralBaseAlgorithm.H>
+
+/* \brief Class that updates the field in spectral space
+ * and stores the coefficients of the corresponding update equation.
+ */
+class GalileanAlgorithm : public SpectralBaseAlgorithm
+{
+    public:
+        GalileanAlgorithm(const SpectralKSpace& spectral_kspace,
+                         const amrex::DistributionMapping& dm,
+                         const int norder_x, const int norder_y,
+                         const int norder_z, const bool nodal,
+                         const amrex::Array<amrex::Real,3>& v_galilean,
+                         const amrex::Real dt);
+        // Redefine update equation from base class
+        virtual void pushSpectralFields(SpectralFieldData& f) const override final;
+        virtual int getRequiredNumberOfFields() const override final {
```

Comment:
please add a Doxygen string that describes the parameters and return value of `getRequiredNumberOfFields`.